### PR TITLE
Removing unused code

### PR DESCRIPTION
**Pull request by Theoklitos**
Removed:

1. The old java library
2. The (unused and ignored) regression tests
3. Faucet
4. java-examples
4. Most of the groovy code
5. The docker and cluster tests. They have been disabled for weeks.

### CODE REVIEW

File: radixdlt-regression/system-tests/src/test/java/com/radixdlt/test/checks/CounterCheck.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
+// * (C) Copyright 2020 Radix DLT Ltd
```
Comment (siy) [inline radixdlt-regression/system-tests/src/test/java/com/radixdlt/test/checks/CounterCheck.java:1]: Perhaps just remove it?

File: radixdlt-regression/system-tests/src/test/java/com/radixdlt/test/checks/LivenessCheck.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
+///*
```
Comment (siy) [inline radixdlt-regression/system-tests/src/test/java/com/radixdlt/test/checks/LivenessCheck.java:1]: Perhaps just remove it?

File: radixdlt-regression/system-tests/src/test/java/com/radixdlt/test/checks/ResponsivenessCheck.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
+///*
```
Comment (siy) [inline radixdlt-regression/system-tests/src/test/java/com/radixdlt/test/checks/ResponsivenessCheck.java:1]: Perhaps just remove it?

File: radixdlt-regression/system-tests/src/test/java/com/radixdlt/test/checks/SystemCounters.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
+///*
```
Comment (siy) [inline radixdlt-regression/system-tests/src/test/java/com/radixdlt/test/checks/SystemCounters.java:1]: Perhaps just remove it?

**Review by siy — APPROVED** [review]
LGTM with minor comments

**Review by LukasGasior1 — APPROVED** [review]

**Pull request merged**